apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/server/RandomFissClaimGenerator.java

```diff
@@ -369,6 +369,7 @@ private void addRandomAudits(FissClaim.Builder claim) {
       optional(() -> payer.setBadtOperId(randomAlphaNumeric(1, 9)));
       optional(() -> payer.setBadtReas(randomAlphaNumeric(1, 5)));
       optional(() -> payer.setBadtCurrDateCymd(randomDate()));
+      payer.setRdaPosition(i);
```

Comment:
@KyRobbins would you mind changing this variable name to something like `audit` or `trail`.  I know you didn't introduce it, but I think it would make the function read a bit better.